PerlLib/GFF3_utils.pm

```diff
@@ -55,7 +55,7 @@ sub index_GFF3_gene_objs {
 
         my @x = split (/\t/);
 
-		unless (scalar @x >= 9) {
+		unless (@x == 9) {
```

Comment:
sometimes the gff3 files embed extra columns for metadata.  I'm not sure this change is critical so will leave it be for now.